packages/nextjs/app/builders/0xb216270aFB9DfcD611AFAf785cEB38250863F2C9/page.tsx

```diff
@@ -0,0 +1,72 @@
+export default function Page() {
+  return (
+    <div className="flex flex-col items-center min-h-screen bg-gradient-to-b from-base-200 to-base-300 p-8">
+      <div className="max-w-3xl w-full bg-gradient-to-r from-pink-50 via-white to-purple-50 shadow-2xl rounded-2xl p-10 border border-pink-100 hover:shadow-pink-200/20 transition-all duration-300">
+        <div className="flex flex-col items-center gap-6">
+          {/* Avatar  */}
+          <div className="relative group">
+            <div className="absolute -inset-0.5 bg-gradient-to-r from-pink-600 to-purple-600 rounded-full opacity-60 group-hover:opacity-100 transition duration-500"></div>
+            <div className="relative w-36 h-36 rounded-full ring-2 ring-white ring-offset-4 ring-offset-pink-50 shadow-md overflow-hidden hover:scale-105 transition-all duration-300">
+              <svg width="" height="" viewBox="0 0 120 120" xmlns="http://www.w3.org/2000/svg">
+                <circle cx="60" cy="60" r="60" fill="#F9D5D3" />
+                <circle cx="60" cy="45" r="20" fill="#FFE0BD" />
+                <path d="M40 90c0-15 40-15 40 0v5H40v-5z" fill="#FFE0BD" />
+                <path d="M40 50c0-10 10-20 20-20s20 10 20 20-10 20-20 20-20-10-20-20z" fill="#3E2723" />
+                <path d="M60 65c10 0 20 10 20 25H40c0-15 10-25 20-25z" fill="#F48FB1" />
+                <circle cx="50" cy="42" r="3" fill="#000" />
+                <circle cx="70" cy="42" r="3" fill="#000" />
+                <path d="M52 50c2 2 6 2 8 0" stroke="#000" stroke-width="1.5" fill="none" />
+              </svg>
+            </div>
+          </div>
+
+          {/* Name */}
+          <div className="space-y-3 text-center">
+            <h1 className="text-5xl font-extrabold bg-gradient-to-r from-pink-600 to-purple-600 text-transparent bg-clip-text hover:scale-105 transition-transform cursor-default">
+              0xRampop
+            </h1>
+            <p className="text-lg text-gray-600 font-medium max-w-md">
+              ✨ Web3 Fullstack Developer | BuidlGuidl Builder | Technical Writer | DeFi Enthusiast ✨
+            </p>
+          </div>
+
+          {/* Address with copy functionality */}
+          <div className="group relative">
+            <div className="text-sm font-mono bg-white/10 text-black px-6 py-3 rounded-xl shadow-inner border border-gray-200 hover:bg-white transition-colors cursor-copy">
+              0xb216270aFB9DfcD611AFAf785cEB38250863F2C9
```

Comment:
There's a component in this project that renders the address. You should try using it.